s3torchconnector/pyproject.toml

```diff
@@ -4,7 +4,7 @@ build-backend = "setuptools.build_meta"
 
 [project]
 name = "s3torchconnector"
-version = "1.0.0"
+version = "1.1.0"
```

Comment:
Need to also change the dependency on `s3torchconnectorclient`'s version

Reply:
Missed that, thanks, updated!